en/docs/deploy/aws.md

```diff
@@ -1,36 +1,177 @@
+---
 
-## Step 1 - Checkout this repository into your local machine using the following Git command
+# Deploying WSO2 Identity Server using AWS CloudFormation
 
-```java
-git clone https://github.com/wso2/aws-is.git
+## Prerequisites
+
+1. Make sure you already have an [AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/). 
+
+2. Install [AWS CLI 2](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html). Alternatively, you can also use AWS CLI version 1. However, you need to make sure that you have installed Python3 to use AWS CLI version 1.
+
+## Step 1 - Create and upload an SSL certificate into AWS
+In AWS, web servers are fronted with a Load balancer. While deploying the WSO2 Identity Server in AWS, we need not create the load balancer separately since it is taken care of in the template we use. For a secure connection via the load balancer between the web server and the browser, we need an SSL certificate. 
+
+??? note "Creating a self signed certificate"
+	If you are using this for testing purposes and do not want to create a certificate using AWS certificate manager, you can create a self signed certificate instead by following the instructions given below. 
+
+	1. Generate private key as private.pem 
+
+		**Request**
+
+		```curl 
+		openssl genrsa -out private.pem 2048
+		```
+		
+		**Response**
+
+		```curl
+		Generating RSA private key, 2048 bit long modulus
+		...................................................................................+++
+		......................................................................................+++
+		e is 65537 (0x010001)
+		```
+
+	2. Generate public key as public.pem
+
+		**Request**
+
+		```curl 
+		openssl rsa -in private.pem -outform PEM -pubout -out public.pem
+		```
+
+		**Response**
+
+		```curl
+		writing RSA key
+		```
+	3. Create a CSR (Certificate Signing Request) as certificate.csr
+
+		**Request**
+
+		```curl
+		openssl req -new -key private.pem -out certificate.csr
+		```
+
+		**Response**
+
+		```curl
+		You are about to be asked to enter information that will be incorporated
+		into your certificate request.
+		What you are about to enter is what is called a Distinguished Name or a DN.
+		There are quite a few fields but you can leave some blank
+		For some fields there will be a default value,
+		If you enter '.', the field will be left blank.
+		-----
+		Country Name (2 letter code) [AU]:
+		State or Province Name (full name) [Some-State]:
+		Locality Name (eg, city) []:
+		Organization Name (eg, company) [Internet Widgits Pty Ltd]:
+		Organizational Unit Name (eg, section) []:
+		Common Name (e.g. server FQDN or YOUR name) []:*us-east-2.elb.amazonaws.com
+		Email Address []:
+		Please enter the following 'extra' attributes
+		to be sent with your certificate request
+		A challenge password []:
+		An optional company name []:
+		```
+	4. Create a Self-signed certificate as certificate.crt
```

Comment:
Self-signed --> self-signed

Is there a reason why the S is capitalized there?